iconv-lite-umd.d.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ *  REQUIREMENT: This definition is dependent on the @types/node definition.
+ *  Install with `npm install @types/node --save-dev`
+ *--------------------------------------------------------------------------------------------*/
+
+declare module "iconv-lite-umd" {
+  // Basic API
+  export function decode(
+    buffer: Buffer,
+    encoding: string,
+    options?: Options
+  ): string;
+
+  export function encode(
+    content: string,
+    encoding: string,
+    options?: Options
+  ): Buffer;
+
+  export function encodingExists(encoding: string): boolean;
+
+  // WARNING: Excluded because it is specific to node.
+
+  // export function decodeStream(
+  //   encoding: string,
+  //   options?: Options
+  // ): NodeJS.ReadWriteStream;
+
+  // export function encodeStream(
+  //   encoding: string,
+  //   options?: Options
+  // ): NodeJS.ReadWriteStream;
```

Comment:
I was thinking it'll be good to leave it commented instead of removing. The idea was to seek for attention of the future contributors and reviewers, so this does not get accidentally contributed back. It'll be much harder to leave such changes unnoticed.

Reply:
OK sure.